docs/source/apis/api_leveling.md

```diff
@@ -0,0 +1,57 @@
+# Llama Stack API Stability Leveling
+
+In order to provide a stable experience in Llama Stack, the various APIs need different stability levels indicating the level of support, backwards compatability, and overall production readiness.
+
+## Different Levels
+
+### v1alpha
+
+- Little to no expectation of support between versions
+- Breaking changes are permitted
+- Datatypes and parameters can break
+- Routes can be added and removed
+
+### v1beta
+
+- API routes ensured between versions
+- Parameters and return types are not
+- API, besides minor fixes and adjustments, should be _almost_ v1. Changes should not be drastic.
+
+### v1 (stable)
+
+- Considered stable
+- Backwards compatible between Z-streams
+  - Y-stream breaking changes must go through the proper approval and announcement process.
+- Datatypes for a route and its return types cannot changed between Z-streams
+  - Y-stream datatype changes should be sparing, unless the changes are additional net-new parameters
+- Must have proper conformance testing as outlined in https://github.com/llamastack/llama-stack/issues/3237
+
+
+### Approval and Announcement Process for Breaking Changes
+
+- **PR Labeling**: Any pull request that introduces a breaking API change must be clearly labeled with `breaking-change`.
+- **Changelog Entry**: The PR must also update the changelog to describe the change, its impact, and any migration steps.
```

Comment:
That's a dup of the "Announcement" section where we include release note already.

Reply:
Announcement through the CHANGELOG. We don't need to separate entries. We can remove the CHANGELOG piece and indicate in the Announcement that the CHANGELOG must reflect the breaking change in a dedicated section.

Reply:
Still not addressed, also the CHANGELOG is typically updated async today so this won't quite work unless we change that process.